client/tailscale/localclient.go

```diff
@@ -778,6 +778,11 @@ func (lc *LocalClient) SetDNS(ctx context.Context, name, value string) error {
 //
 // The ctx is only used for the duration of the call, not the lifetime of the net.Conn.
 func (lc *LocalClient) DialTCP(ctx context.Context, host string, port uint16) (net.Conn, error) {
+	return lc.UserDial(ctx, "tcp", host, port)
+}
+
+// UserDial is like DialTCP, but allows specifying the network type.
```

Comment:
this could use a real comment.

As is it reads like "This is like TCP but not TCP, so maybe that means tcp4/tcp6 or maybe it means udp or sctp dunno"

But it returns a net.Conn so I'll assume it's not UDP.

Reply:
`net.Dial` also returns a `net.Conn` but it works with UDP fine.
https://pkg.go.dev/net#Dial

Reply:
Updated comment though.